.github/workflows/mkosi.yml

```diff
@@ -197,7 +197,7 @@ jobs:
 
     - name: Archive packages
       uses: actions/upload-artifact@v4
-      if: success() || failure()
+      if: (success() || failure()) && github.repository == 'systemd/systemd' || github.repository == 'systemd/systemd-stable'
```

Comment:
ditto

Reply:
Also, maybe you can just use `always()` instead of `success() || failure()`?

Reply:
@mrc0mmand `always()` means when cancelled as well, and we don't want that.